command/helper/config.go

```diff
@@ -14,26 +14,24 @@ import (
 	"github.com/0xPolygon/polygon-sdk/server"
 	"github.com/0xPolygon/polygon-sdk/types"
 	"github.com/hashicorp/hcl"
-	"github.com/imdario/mergo"
 )
 
 // Config defines the server configuration params
 type Config struct {
-	Chain          string                        `json:"chain"`
-	DataDir        string                        `json:"data_dir"`
-	BlockGasTarget string                        `json:"block_gas_target"`
-	GRPCAddr       string                        `json:"rpc_addr"`
-	JSONRPCAddr    string                        `json:"jsonrpc_addr"`
-	Telemetry      *Telemetry                    `json:"telemetry"`
-	Network        *Network                      `json:"network"`
-	SecretsManager *secrets.SecretsManagerConfig `json:"secrets_manager"`
-	Seal           bool                          `json:"seal"`
-	TxPool         *TxPool                       `json:"tx_pool"`
-	LogLevel       string                        `json:"log_level"`
-	Consensus      map[string]interface{}        `json:"consensus"`
```

Comment:
What happened to this consensus configuration?

We are using it for some pending PRs, namely the PoA / PoS implementation

Reply:
I couldn't find any use in current implementation, Restored it [here](https://github.com/0xPolygon/polygon-sdk/pull/270/commits/562271a4e44449e9c3dfe1fd1c31df0401ea8407)